Vagrantfile

```diff
@@ -256,7 +257,9 @@ Vagrant.configure("2") do |config|
           ansible.host_key_checking = false
           ansible.raw_arguments = ["--forks=#{$num_instances}", "--flush-cache", "-e ansible_become_pass=vagrant"]
           ansible.host_vars = host_vars
-          ansible.tags = ['facts']
+          if $ansible_tags
```

Comment:
```suggestion
          ansible.tags = $ansible_tags||['facts']
```

Reply:
For normal users(not CI), we don't need to specify `ansible.tags`, right?

Reply:
Indeed, I have not considered this aspect.